vortexasdk/client.py

```diff
@@ -205,3 +208,19 @@ def set_client(client) -> None:
     logger.debug(
         f"global __client__ has been set {__client__.__class__.__name__} \n"
     )
+
+
+def warn_user_if_sdk_version_outdated() -> None:
+    """Warn users if their SDK version is outdated"""
+    try:
+        latest_sdk_version, sdk_outdated_check = is_sdk_version_outdated()
+        if sdk_outdated_check:
+            logger.warning(
+                f"You are using {sdk_pkg_name} version {__version__}, however version {latest_sdk_version} is available.\n"
+                f"You should consider upgrading via the 'pip install {sdk_pkg_name} --upgrade' command."
+            )
+    except Exception as e:
+        logger.warning(
+            f"Outdated SDK version check could not be completed. \n"
+            f"Got an exception: {e}"
+        )
```

Comment:
Separate function in the client script to raise a warning if the SDK version is outdated.

Reply:
Nice